ckanext/validation/logic.py

```diff
@@ -661,6 +661,13 @@ def _run_sync_validation(resource_id, local_upload=False, new_resource=True):
 
     if validation['report']:
         report = json.loads(validation['report'])
+    # Check upload of invalid data is allowed
```

Comment:
I don't think this comment provides any extra info than just the code itself. Please remove it.